network/common/src/main/java/org/apache/spark/network/sasl/SaslClientBootstrap.java

```diff
@@ -33,6 +34,8 @@
 import org.apache.spark.network.util.JavaUtils;
 import org.apache.spark.network.util.TransportConf;
 
+import java.util.Properties;
```

Comment:
Please move this line after line21. It's required by https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports
Besides, you could check the code style issue by running script ```sh dev/scalastyle``` under the spark project dir.